images/pf-status-relay.yml

```diff
@@ -0,0 +1,27 @@
+content:
+  source:
+    dockerfile: Dockerfile.openshift
+    git:
+      branch:
+        target: release-{MAJOR}.{MINOR}
+      url: git@github.com:openshift-priv/pf-status-relay.git
+      web: https://github.com/openshift/pf-status-relay
+    ci_alignment:
+      streams_prs:
+        ci_build_root:
+          stream: rhel-9-golang-ci-build-root
+distgit:
+  branch: rhaos-{MAJOR}.{MINOR}-rhel-9
+  component: pf-status-relay-container
+enabled_repos:
+- rhel-9-appstream-rpms
+- rhel-9-baseos-rpms
+for_payload: false
+from:
+  builder:
+  - stream: rhel-9-golang
+  member: openshift-enterprise-base-rhel9
+name: openshift/pf-status-relay-rhel9
+name_in_bundle: pf-status-relay
```

Comment:
I'm guessing we need `name_in_bundle` here since it'll be consumed by an operator?

Reply:
Operator will be developed in 4.18.

Reply:
If the operator that consumes this will be in 4.18, then should this be a 4.18 component add?

Reply:
Will add ```name_in_bundle: pf-status-relay``` to 4.18 PR